### PR TITLE
Add the Art of Monitoring episode

### LINKED ISSUE #256: Create episode page for The Art of Monitoring
This is episode 74.

Do not merge until @joelaha has uploaded the audio.

Depends on #255

### PR DESCRIPTION
**Pull request by mattstratton**
fixes #256 

Do not merge until @joelaha uploads audio and @trevorghess publishes DevOpsDays DFW episode.

Confirm the date (currently set for 10/1) for publishing.

### CODE REVIEW

**Comment by mattstratton** [comment]
So because of something weird in our config, I can't really test the shorter redirect with the deploy preview, but it's Probably Fine™.

File: data/guests/jturnbull2.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Name: jturnbull
+"FullName": "James Turnbull"
+"Bio": "James Turnbull is the author of seven technical books about open source software, including a book on Docker. He was previously the CTO at Kickstarter and in leadership roles at Docker, Venmo and Puppet. James speaks regularly at conferences including OSCON, Velocity, Linux.conf.au, FOSDEM and a number of others. He is a past president of Linux Australia, a former committee member of Linux Victoria, was Treasurer for LCA 2008, and serves on the program committees of LCA and OSCON."
```
Comment (jamtur01) [inline data/guests/jturnbull2.yml]: and serves as the co-chair of O'Reilly's Velocity conference and on the program committees of Linux.conf.au and OSCON.

Comment (mattstratton) [inline data/guests/jturnbull2.yml]: Better? :)

File: content/episode/art-of-monitoring-james-turnbull.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
++++
+Description = "Friend of the podcast James Turbull joins us to talk about his new book, The Art of Monitoring...and a little bit about this whole functional programming thing"
```
Comment (jamtur01) [inline content/episode/art-of-monitoring-james-turnbull.md]: Turnbull

Comment (mattstratton) [inline content/episode/art-of-monitoring-james-turnbull.md]: Fixed. Whoops!

File: data/guests/jturnbull2.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Name: jturnbull
+"FullName": "James Turnbull"
+"Bio": "James Turnbull is the author of seven technical books about open source software, including a book on Docker. He was previously the CTO at Kickstarter and in leadership roles at Docker, Venmo and Puppet. James speaks regularly at conferences including OSCON, Velocity, Linux.conf.au, FOSDEM and a number of others. He is a past president of Linux Australia, a former committee member of Linux Victoria, was Treasurer for LCA 2008, and serves on the program committees of LCA and OSCON. James serves as the co-chair of O'Reilly's Velocity conference and on the program committees of Linux.conf.au and OSCON."
```
Comment (jamtur01) [inline data/guests/jturnbull2.yml]: eight technical books

Comment (jamtur01) [inline data/guests/jturnbull2.yml]: Sorry! One more - this can be deleted: ", and serves on the program committees of LCA and OSCON."

Comment (mattstratton) [inline data/guests/jturnbull2.yml]: fixed 🚀

Comment (mattstratton) [inline data/guests/jturnbull2.yml]: Fixed this one too 🍰

File: content/episode/art-of-monitoring-james-turnbull.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+author = "Matt"
+categories = []
+date = "2016-10-01T20:54:48-05:00"
+episode = "74"
+friendly = "art-of-monitoring-james-turnbull"
+guests = ["jturnbull2"]
+images = ["http://arresteddevops.com/img/social/fb/art-of-monitoring-james-turnbull.png"]
+news_keywords = []
+podcast = "https://media.blubrry.com/arresteddevops/content.blubrry.com/arresteddevops/arrested-devops-podcast-episode074.mp3"
+podcast_duration = "1:17:48"
```
Comment (joelaha) [inline content/episode/art-of-monitoring-james-turnbull.md]: Episode length is actually 1:10:39

**Comment by joelaha** [comment]
Episode uploaded, @mattstratton.

**Pull request merged**